files/es/web/javascript/reference/global_objects/infinity/index.md

```diff
@@ -10,21 +10,17 @@ original_slug: Web/JavaScript/Referencia/Objetos_globales/Infinity
 
 La propiedad global **`Infinity`** es un valor numérico que representa el infinito.
 
-{{js_property_attributes(0,0,0)}}
+## Valor
 
-## Sintaxis
+Lo mismo que el valor de {{JSXref("Number.POSITIVE_INFINITY")}}.
 
-```js
-Infinity
-```
+{{js_property_attributes(0,0,0)}}
 
-## Description
+## Descripción
 
 `Infinity` es una del objeto global, es decir, esta es una variable de alcance global.
 
-El valor inicial de `Infinity` es {{jsxref("Number.POSITIVE_INFINITY")}}. El valor de `Infinity` (infinito positivo) es mayor que cualquier otro numero incluido el mismo. Este valor se comporta matematicamente como el infinito matemático; por ejemplo, cualquier numero multiplicado por `Infinity` da como resultado `Infinity`, y cualquier numero divido por `Infinity` da como resultado cero.
-
-Para la especificación ECMAScript 5, `Infinity` es de solo lectura (implementado en JavaScript 1.8.5 / Firefox 4).
+El valor inicial de `Infinity` es {{jsxref("Number.POSITIVE_INFINITY")}}. El valor de `Infinity` (infinito positivo) es mayor que cualquier otro numero. Este valor se comporta matematicamente como el infinito matemático; por ejemplo, cualquier numero multiplicado por `Infinity` da como resultado `Infinity`, y cualquier numero divido por `Infinity` da como resultado cero.
```

Comment:
The last sentence here, i.e. "Este valor se comporta… cero.", should also be syncced with en-US, but that's far beyond my poor Spanish :/

Reply:
Thank you so much for pointing it! @yarusome 

I'm going to use this opportunity to sync the whole article